pkgs/os-specific/darwin/adv_cmds/locale.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl }:
```

Comment:
This one seems worth merging into https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/apple-source-releases/adv_cmds/default.nix.

I also noticed another duplicate in https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/adv_cmds/ps.nix. Not sure how that reappeared. I was probably just sloppy when I merged in the apple source releases.